code-challenges.test.js

```diff
@@ -17,28 +17,70 @@
 
 // a) Create a test with expect statements for each of the variables provided.
 
-var num1 = 15
-// Expected output: "15 is divisible by three"
 
-var num2 = 0
+describe("when evenlyDivisible is called"); => {
+
+  it("returns '15 is divisble by three when called with 15'", () => {
+    //arrange
+    const argument = 15
+    //act
+    const actualDivisable = evenlyDivisible(argument);
+    //assert
+    expect(actualDivisable).toequal("15 is divisable by three")
+  });
+//Expected output: "15 is divisible by three"
+
+  it("returns '0 is divisble by three' when called with 0", () => {
+    //arrange
+    const argument = 0
+    //act
+    const actualDivisable = evenlyDivisible(argument);
+    //assert
+    expect(actualDivisable).toequal("0 is divisble by three");
+  });
 // Expected output: "0 is divisible by three"
 
-var num3 = -7
-// Expected output: "-7 is not divisble by three"
+  it("returns '-7 is not divisable by 3' when called with -7", () => {
+    //arrange
+    const argument = -7
+    //act
+    const actualDivisable = evenlyDivisible(argument);
+    //assert
+    expect(actualDivisable).toequal("-7 is divisable by three");
+  });
 
+// Expected output: "-7 is not divisble by three"
+// b) Create the function that makes the test pass.
+var num1 = 15
+var num2 = 0
+var num3 = -7
 
 
+if(num1 % 2) {
```

Comment:
Looks like you'll need to create a function that holds this logic in order for your tests to run properly. It looks like in your tests, you have `evenlyDivisible` as the function name so you can start with using that as the function name for this logic. 

This logic is referencing the variables `num1` `num2` and `num3` directly which with a function you'll need to use an argument instead.